css/project-page.styl

```diff
@@ -59,13 +59,58 @@
   display: flex
   flex-direction: column
   flex: 1 0 auto
+  font-size: 14px
 
   .call-to-action-container
-    align-items: center;
+    min-height: 500px
+    align-items: center
     display flex
     flex: 1 0 auto
     flex-direction: column
-    justify-content: center;
+    justify-content: center
+    max-width: 800px
+    margin: 0 auto
 
   .description
-    font-size: 1.5em
+    font-size: 2.2em
+    font-weight: 700
+    line-height: 1.5em
+    text-align: center
+    margin: 50px 0 40px 0
+
+  .workflow-description
+    line-height: 1.5em
+    margin: 10px 0 30px 0
+    
+  .standard-button
+    margin-top: 10px
+    font-weight: 700
+    line-height: 1.7em
+
+  hr
+    margin: 50px 0 0 0
+
+  .introduction
+    background: #fff
+    margin: 0
+    padding: 1.5em 3vw
+    width: 100%
+    box-sizing: border-box
+
+    .about-project
+      text-transform: uppercase
+      font-weight: 700
+      line-height: 2.5em
+      font-size: 15px
+      color: #5a5a5a
+      text-align: center
+      max-width: 800px
+      margin: 0 auto
+
+    p
+      color: #989898
+      font-size: 1em
+      line-height: 1em
```

Comment:
These `p`s look a bit cramped with multiple lines. I'd say bump this line-height up a bit or remove this line and then it's good to go :+1: